ui/src/PaperSummary.tsx

```diff
@@ -23,6 +23,11 @@ function warnOfUnimplementedActionAndTrack(actionType: string) {
   }
 }
 
+function goToLibrary() {
+  const s2LibraryUrl = `https://www.semanticscholar.org/me/library`;
```

Comment:
we should add a utm source here so we know where traffic is coming from to S2's library